modules/price/commerce_price.services.yml

```diff
@@ -20,3 +20,9 @@ services:
     class: Drupal\commerce_price\Resolver\DefaultPriceResolver
     tags:
       - { name: commerce_price.price_resolver, priority: -100 }
+
+  commerce_price.twig_extension:
+      arguments: ['@renderer']
```

Comment:
No need to inject the render once we stop inheriting from the core twig extension class.